content/blogposts/2023/release-4.5.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Sourcegraph 4.5 release"
+publishDate: 2023-02-22T10:00-07:00
+description: "Sourcegraph 4.5 introduces..."
+tags: [blog, release]
+slug: "release/4.5"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.5/sourcegraph-4-5-hero.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.5/sourcegraph-4-5-hero.png
+changelogItems:
+  - description: "Zoekt now bypasses the regex engine for common queries, such as `\\bLITERAL\\b case:yes`. This can lead to a significant speed-up for \"Find references\" and \"Find implementations\" if precise code navigation is not available."
+    url: https://github.com/sourcegraph/zoekt/pull/526
+    category: Search
+  - description: "Code Insights has a new display option: \"Max number of series points to display\". This setting controls the number of data points you see per series on an insight."
+    url: https://github.com/sourcegraph/sourcegraph/pull/46653
+    category: Code Insights
+  - description: "You can now export all data for a code insight from the card menu or the standalone page."
+    url: https://github.com/sourcegraph/sourcegraph/pull/46694
+    category: Code Insights
+- description: Added Gerrit as an officially supported code host with permissions syncing.
+    url: https://github.com/sourcegraph/sourcegraph/pull/46763
+    category: Admin
+  - description: "Monitoring: the searcher dashboard now contains more detailed request metrics as well as information on interactions with the local cache (via gitserver)."
+    url: https://github.com/sourcegraph/sourcegraph/pull/47654
+    category: Admin
+  - description: "Zoekt has a new opt-in feature, \"shard merging\". Shard merging consolidates small index files into larger ones, which reduces Zoekt-webserver's memory footprint, especially for users with many small and rarely update repositories."
+    url: https://docs.sourcegraph.com/code_search/explanations/search_details#shard-merging
+    category: Admin
+---
+
+<Badge link="https://docs.sourcegraph.com/code_insights" text="Code Insights" color="green" size="small" />
+
+#### The new Code Insights UI makes it easier to run insights over specific repositories
```

Comment:
@chwarwick , could you please review this section? Is this accurate?

I am also planning to add a GIF to this shortly.

Reply:
@iskyOS 

>  Previously, repositories had to be explicitly listed.This change makes it easier to select very specific and large repository groups to run insights over.

This is all true, but we want to key on this being about picking smaller sets of repos while stilling getting all the data you are looking for. I'm having a hard time coming up with the words for this but something along the lines of.

``` 
Previously, if you wanted more than a few repositories you had to run your insight over all of them.  This change makes it possible to select only the repositories that are relevant to the insight you are creating. 
```

The background reason for this change is that for customer with many repos running an insight over all of them simply takes too long and can accumulate very large amounts of data.  This change makes it possible for the user to pick the **relevant** repos, somewhere greater than 10 but less than all; so that users can keep the size of the insight down but still get all the important information.  Some examples could be all repos that follow a naming convention (like `infrastructure`), or all repos that have a `package.json` file.

> The new query system allows you to select repositories with these filters:

Maybe just a small change here
```
The new system allows you to use any of the `repo:` filter options such as:  
- `repo:`
- `-repo:`
- `repo:has.path()`
- `repo:has.file()`
- `repo:has.commit.after()`
```

The `repo:.*` is "all repos" so I don't want to encourage the use of that one.